psalm.xml

```diff
@@ -4,9 +4,12 @@
         phpVersion="8.1"
         resolveFromConfigFile="true"
         rememberPropertyAssignmentsAfterCall="false"
+        ensureOverrideAttribute="false"
```

Comment:
Please set this to true, override attributes are recommended adn compatible with all PHP versions if the symfony/polyfill-php83 packages required